buildSrc/src/main/java/io/micronaut/guides/core/App.java

```diff
@@ -0,0 +1,54 @@
+package io.micronaut.guides.core;
+
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
+import io.micronaut.core.annotation.NonNull;
+import io.micronaut.core.annotation.Nullable;
+import io.micronaut.jsonschema.JsonSchema;
+import io.micronaut.serde.annotation.Serdeable;
+import io.micronaut.starter.application.ApplicationType;
+
+import java.util.List;
+
+@JsonSchema
+@Serdeable
+public record App (
```

Comment:
Populate the description via javadoc. see: 

https://github.com/micronaut-projects/micronaut-guides/blob/master/guides/micronaut-json-schema/java/src/main/java/example/micronaut/Product.java#L26-L32